eng/push-tarball.sh

```diff
@@ -129,6 +129,16 @@ pushd "$vmr_path"
     git push -u upstream "$new_branch_name"
 
     echo "Creating PR from $new_branch_name to $target_branch"
-    curl -H 'Content-Type: application/json' -d "$data" "$pr_url"
+    curlResult=$(curl -H 'Content-Type: application/json' -d "$data" "$pr_url" 2>/dev/null)
+    webUrl=$(jq -r '.repository.webUrl | values' <<< "$curlResult")
+    pullRequestId=$(jq -r '.pullRequestId | values' <<< "$curlResult")
+    if [ -n "$webUrl" ] && [ -n "$pullRequestId" ]; then
+      echo "$webUrl"/pullrequest/"$pullRequestId"
+    elif [[ "$curlResult" == *"TF401179"* ]]; then
```

Comment:
1) Where does this magical constant come from? It would be good to include a brief explanation in a comment so that readers of this code understand where does this come from.

2) Using the glob here is a bit strange as you are efficiently matching the string. Recommend to use the =~ operator for regexp match instead, i.e., `[[ "$curl_result" =~ "TF401179" ]]`